.github/workflows/assign_to_project.yml

```diff
@@ -0,0 +1,44 @@
+name: Assign to one project
+
+on:
+  issues:
+    types: [opened, labeled, unlabeled]
+  pull_request:
+    types: [opened, labeled, unlabeled]
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  assign_one_project:
+    runs-on: ubuntu-latest
+    name: Assign to One Project
+    steps:
+    - name: Assign new and labeled issues/pull-requests to project code backlog
+      uses: srggrs/assign-one-project-github-action@1.2.0
+      if: |
+        github.event.action == 'opened' &&
+        (contains(github.event.issue.labels.*.name, 'bug') ||
+        contains(github.event.pull_request.labels.*.name, 'bug') ||
+        contains(github.event.issue.labels.*.name, 'dependencies') ||
+        contains(github.event.pull_request.labels.*.name, 'dependencies') ||
+        contains(github.event.issue.labels.*.name, 'documentation') ||
+        contains(github.event.pull_request.labels.*.name, 'documentation') ||
+        contains(github.event.issue.labels.*.name, 'duplicate') ||
+        contains(github.event.pull_request.labels.*.name, 'duplicate') ||
+        contains(github.event.issue.labels.*.name, 'enhancement') ||
+        contains(github.event.pull_request.labels.*.name, 'enhancement') ||
+        contains(github.event.issue.labels.*.name, 'feature') ||
+        contains(github.event.pull_request.labels.*.name, 'feature') ||
+        contains(github.event.issue.labels.*.name, 'hot') ||
+        contains(github.event.pull_request.labels.*.name, 'hot'))
+      with:
+        project: 'https://github.com/pyfar/pyfar/projects/2'
+
+    - name: Assign new and labeled issues/pull-requests to project general
+      uses: srggrs/assign-one-project-github-action@1.2.0
+      if: |
+        github.event.action == 'opened' &&
+        (contains(github.event.issue.labels.*.name, 'question') ||
+        contains(github.event.pull_request.labels.*.name, 'question'))
```

Comment:
Yes, I think in this case it will be assigned to both projects and anything labeled question will be moved to *General*, however, it is always possible to manually move things around after they were automatically assigned. I wanted to keep it simple, but we could have more cases if we want...

Reply:
I'm also for not overengineering it.

Reply:
+1 for keeping it simple. :)